webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+
+module.exports = {
+  mode: 'production',
+  entry: './src/exports.js',
+  output: {
+    filename: 'main.js',
```

Comment:
Does the output filename need to match the `browser` file in `package.json`? Right now they appear to be different.

Reply:
_Excellent_ catch. I updated the field.